### PR TITLE
fix: LEAP-182: Org and deployment level setting for cert verification

### PR DESCRIPTION
**Pull request by jombooth**
### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [x] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

In general, it's safer to verify SSL certificates when making external requests, so do that by default. For clients that can't support this, make it possible to turn off SSL certificate verification deployment-wide via settings.VERIFY_SSL_CERTS, and also make it possible to override that setting at the organization level in Django Admin, for use in the uploader and similar surfaces.

Also add WINDOWS_SQLITE_BINARY_HOST_PREFIX to base.py, to facilitate specifying a host with a valid certificate if necessary. Currently the host we download these binaries from is working fine, but historically we've had issues with the cert not verifying.

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

**Review by makseq — APPROVED** [review]

File: label_studio/organizations/models.py [hunk 1]
```diff
@@ -141,6 +148,11 @@ def per_project_invited_users(self):
         per_project_invited_users = User.objects.filter(pk__in=invited_ids)
         return per_project_invited_users
 
+    def should_verify_ssl_certs(self) -> bool:
```
Comment (makseq) [inline label_studio/organizations/models.py:144]: we have similar functionality in licenses: 
https://github.com/HumanSignal/label-studio-enterprise/blob/develop/label_studio_enterprise/billing/models.py#L45

better to use the same mechanics

File: label_studio/core/settings/base.py [hunk 2]
```diff
@@ -108,6 +108,14 @@
 DEBUG = get_bool_env('DEBUG', True)
 DEBUG_MODAL_EXCEPTIONS = get_bool_env('DEBUG_MODAL_EXCEPTIONS', True)
 
+# Whether to verify SSL certs when making external requests, eg in the uploader
+# ⚠️ Turning this off means assuming risk. ⚠️
+# Overridable at organization level via Organization#verify_ssl_certs
+VERIFY_SSL_CERTS = get_bool_env('VERIFY_SSL_CERTS', True)
```
Comment (makseq) [inline label_studio/core/settings/base.py:114]: Please let our customers know about this VERIFY_SSL_CERTS var before they roll out this version

File: label_studio/organizations/models.py (hunk 1 above)
Comment (jombooth) [reply label_studio/organizations/models.py:144]: agreed, I'll remove the Organization field and migration, great call here :+1:

File: label_studio/core/settings/base.py (hunk 2 above)
Comment (jombooth) [reply label_studio/core/settings/base.py:114]: :100:

**Pull request merged**